.github/workflows/test-common-tooling.yml

```
@@ -0,0 +1,77 @@
+---
+name: Test Common Tooling
+
+on:
+  pull_request:
+    paths:
+    - 'common-tooling/action.yml'
+
+jobs:
+  test-common-tooling:
+    strategy:
+      matrix:
+        label: ["ubuntu-latest", "gcp-core-2-default"]
+    runs-on: ${{ matrix.label }}
+    timeout-minutes: 10
+    steps:
+    - uses: actions/checkout@v3
+    - uses: ./common-tooling
```

I suggest giving the steps the human-readable names for identification as well, e.g.:
```suggestion
    - uses: ./common-tooling
      name: Install defaults and configure maven-settings.xml
```

---

Though I guess it's not very usable in the logs now (https://github.com/actions/runner/issues/1877) still adding a bit readability be good.